Issue4~6の解消

yukimatsuri-01 opened the pull request:
# 概要  
Issue4~6の解消  
アプリケーションのカレンダーの画面↓
https://gyazo.com/8bbc7f262fb4b62f6649a59a0ffb20ec

yukimatsuri-01 committed:
Issue4

yukimatsuri-01 committed:
Issue5

yukimatsuri-01 committed:
Issue6

yukimatsuri-01 committed:
Merge branch 'master' into second_training

chirikuzu commented on app/controllers/calendars_controller.rb:
```diff
@@ -34,7 +34,11 @@ def get_week
+      if wday_num == 7
+        wday_num = wday_num - 7
+      end
+      days = { month: (@todays_date + x).month, date: (@todays_date+x).day, plans: today_plans, wday: (@todays_date + x).wday}
```
こちらですが、キャプチャを確認させていただいたところ曜日の表示が正常に行えていないようです。
カリキュラムを再度確認し、曜日の表示が正常に行えるように修正を行いましょう。
わからないことがあればコネクトにて質問してみましょう。
https://master.tech-camp.in/curriculums/4682

yukimatsuri-01 replied:
修正しました。

chirikuzu reviewed (commented):
レビュー依頼ありがとうございます！
コメントしましたので、確認お願い致します！

ghost commented:
レビュー依頼ありがとうございます。
こちら修正差分がgithub上に上がっていないようです。
コミット、プッシュのし忘れなどをご確認の上、再度コードレビューのご提出をお願いいたします。
また、改めてgyazoによるキャプチャのご提出もお願いいたします。
大変お手数ではございますが、よろしくお願いいたします。

yukimatsuri-01 committed:
Update calendars_controller.rb

大変申し訳ございません。再度修正しましたのでご確認いただけますと幸いです。
https://gyazo.com/364ed47002d1c3350403604092126c3e

yukimatsuri-01 commented:
修正しました。
https://gyazo.com/364ed47002d1c3350403604092126c3e

TOMARUMARU reviewed (commented):
LGTMです。お疲れ様でした。mergeして次に進みましょう。
